Algebra.java

```diff
@@ -25,43 +25,106 @@ public static void main(String args[]) {
 
```

Comment:
There were HW violations in this assignment.
I will clarify again: during the assignment, you are allowed to use only things you were taught in Lectures or Recitations, unless you were specifically told otherwise.

Each of those violation cause deduction of points..
Total deductions: 4 points.

Talk to me,
Tal Danai